### DISCUSSION TITLE
When I use Rustdesk to control other computers, their Rustdesk client appears blank

### DISCUSSION THREAD

**Discussion by TryingToDoBetter25** [rustdesk/rustdesk#9861] [Q&A]
When I use Rustdesk to control other computers, their Rustdesk client appears blank. Why is this? Is there a solution？
I have tried some known methods, but they still don't work
version：rustdesk-1.3.2-x86_64
Windows10pro
![PixPin_2024-11-08_14-04-03](https://github.com/user-attachments/assets/eeb14976-18fb-49f9-a0f5-c21730b4d8f7)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/418#issuecomment-1233024266
